features/cellular/TARGET_GENERIC_MODEM/generic_modem_driver/ReferenceCellularDriver.h

```diff
@@ -0,0 +1,277 @@
+/* Copyright (c) 2017 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef REFERENCE_CELLULAR_DRIVER_
+#define REFERENCE_CELLULAR_DRIVER_
+
+#include "mbed.h"
```

Comment:
mbed.h should include all thats needed here or at least most. Needs to be trimmed back.